package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "react-rainbow-components",
-    "version": "0.6.0",
+    "version": "0.5.1",
```

Comment:
this is incorrect. Are we going back in time? :)

Reply:
sorry a got mixed on the merge